src/service/holdings.ts

```diff
@@ -122,6 +142,10 @@ export async function getHoldingsCelo() {
       celoUnfrozenBalance(),
     ])
   )
+  //TODO:
+  //Uncomment and check once assets are bridged and pools are active
+  //unfrozen.value += await uniV3HoldingsForToken(RESERVE_MULTISIG_CELO, CELO_ADDRESS)
+  //unfrozen.value += await uniV3HoldingsForToken(RESERVE_MULTISIG_CELO, STAKED_CELO_ERC20_ADDRESS)
```

Comment:
I'm unsure if this should be counted here or in the `celoCustodied` (or both), can you ask @rcroessmann for some input here?